shared/constants/network.ts

```diff
@@ -267,10 +271,9 @@ export const TEST_NETWORK_TICKER_MAP: {
   [NETWORK_TYPES.SEPOLIA]: `${typedCapitalize(NETWORK_TYPES.SEPOLIA)}${
     CURRENCY_SYMBOLS.ETH
   }`,
-  [NETWORK_TYPES.LINEA_TESTNET]:
-    `Linea${CURRENCY_SYMBOLS.ETH}` as `${Capitalize<
-      typeof NETWORK_TYPES.LINEA_TESTNET
-    >}${typeof CURRENCY_SYMBOLS.ETH}`,
+  [NETWORK_TYPES.LINEA_GOERLI]: `Linea${CURRENCY_SYMBOLS.ETH}` as `${Capitalize<
```

Comment:
Nit: rather than use a type cast here, we could update the return type to just be `string`. I don't see any purpose in having the type misrepresent the actual value like this - it's misleading and unnecessary.